tests/util/date.js

```diff
@@ -145,6 +146,17 @@ function getDateAndTime(date) {
     return `${date.getDate().toString().padStart(2, '0')}-${date.toLocaleString('en-us', { month: 'short' })}-${date.getFullYear()} ${today.getHours().toString().padStart(2, '0')}:${today.getMinutes().toString().padStart(2, '0')}`
 }
 
+function validateTime(actual,dateTime) {
+    if(actual==dateTime)
+    {
+        return;
+    }
+    let second=dateTime.split(":")[1]
+    second=String(second-1).padStart(2, '0');
+    dateTime=dateTime.split(":")[0]+":"+second;
```

Comment:
parse the date using date library and get the desired output. dont use split & padStart.
what if date and time or not equal? this function will be called recursively?

Reply:
this is not resolved.

Reply:
I have removed date and time validation,check the recent commit